app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <transition enter-from-class="translate-x-full w-0"
+              enter-active-class="transition-all ease-sharp duration-[588ms]"
```

Comment:
why 588ms?

Reply:
That's the same duration as the items card